riscv/insns/cbo_flush.h

```diff
@@ -0,0 +1,11 @@
+require_extension(EXT_ZICBOM);
+if (((STATE.prv != PRV_M) && (get_field(STATE.menvcfg->read(), MENVCFG_CBCFE) == 0)) ||
+    ((STATE.prv == PRV_U && !STATE.v) && (get_field(STATE.senvcfg->read(), SENVCFG_CBCFE) == 0)))
+  throw trap_illegal_instruction(insn.bits());
+else if (((STATE.prv == PRV_S && STATE.v) && (get_field(STATE.henvcfg->read(), HENVCFG_CBCFE) == 0)) ||
+    ((STATE.prv == PRV_U && STATE.v) && ((get_field(STATE.henvcfg->read(), HENVCFG_CBCFE) == 0) ||
```

Comment:
Logic here should be simplified to remove the repeated `(get_field(STATE.henvcfg->read(), HENVCFG_CBCFE) == 0)`. If `STATE.v` then you know `STATE.prv` must be either S or U.

Reply:
OK